packages/mask/src/plugins/MaskPluginWrapper.tsx

```diff
@@ -26,38 +32,96 @@ const useStyles = makeStyles()((theme) => {
                 : null),
         },
         header: {
-            backgroundColor: theme.palette.background.paper,
+            backgroundColor: 'transparent',
             color: theme.palette.text.primary,
             display: 'flex',
             alignItems: 'center',
-            padding: theme.spacing(1, 2),
-            borderBottom: `1px solid ${theme.palette.divider}`,
+            padding: theme.spacing(2),
         },
         title: {
             display: 'flex',
             flexDirection: 'column',
-            paddingLeft: theme.spacing(1),
+            paddingLeft: theme.spacing(1.5),
+        },
+        action: {
+            flex: 1,
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'flex-end',
         },
         body: {
-            margin: theme.spacing(2),
+            borderTop: `1px solid ${theme.palette.divider}`,
+            padding: theme.spacing(2),
+        },
+        button: {
+            color: MaskColorVar.twitterButtonText,
+            '&,&:hover': {
+                background: MaskColorVar.twitterButton,
+            },
         },
     }
 })
 
 export default function MaskPluginWrapper(props: PluginWrapperProps) {
     const { classes } = useStyles()
-    const { pluginName, children } = props
+    const { pluginName, children, action, publisher } = props
+    const personaConnectStatus = usePersonaConnectStatus()
```

Comment:
You should not mix persona connect logic into this component.

If you only want to reuse the UI, please split the UI as a separate component.

Reply:
@yanzhihong23 this one